docs/software/cloud.md

```diff
@@ -442,6 +442,14 @@ Your very own GitHub style server, with web interface.
         - [Fail2Ban](../system_security/#fail2ban-protects-your-system-from-brute-force-attacks)
         - <https://docs.gitea.io/en-us/fail2ban-setup>
 
+=== "Updating"
```

Comment:
We normally named it "Update to latest version".

Reply:
Yes, I named it "Updating" here as it's the same for Firefox Sync Server below. But true, on the other pages it's mostly "Update to latest version". But actually, does it help so much when using the long version? Since software titles will all get an increasing number of doc tabs, having so long tab descriptions make the tabs block become quite large and looking somehow ugly to me. So I personally like single word tabs, like "Updating", "Logging", "Directories", "Tweaking"/"Configuration" etc.

But I'll follow your suggestion for now to align it with other docs pages and we can discuss the tabs naming and order separately.